lib/server.js

```diff
@@ -15,6 +15,8 @@
 	require('./rfs');
 	require('./rs3');
 	require('./wire');
+	require('./azure')
```

Comment:
correct! Tho, until you have it 100% working let's keep it commented out in PRs/prod. If a dev doesn't have Azure optionally NPM installed we want to make sure nothing crashes/breaks.